quickjs.c

```diff
@@ -41116,7 +41116,7 @@ static const JSCFunctionListEntry js_number_funcs[] = {
     JS_CFUNC_DEF("isSafeInteger", 1, js_number_isSafeInteger ),
     JS_PROP_DOUBLE_DEF("MAX_VALUE", 1.7976931348623157e+308, 0 ),
     JS_PROP_DOUBLE_DEF("MIN_VALUE", 5e-324, 0 ),
-    JS_PROP_DOUBLE_DEF("NaN", NAN, 0 ),
+    JS_PROP_U2D_DEF("NaN", 0x7FF8ull<<48, 0 ), // workaround for msvc
```

Comment:
Would it make sense to define JS_FOLAT64_NAN like so?

Reply:
Already exists and defined as follows :)
```c
#define JS_FLOAT64_NAN NAN
```
I initially named it `JS_COMPILE_TIME_NAN` but then I thought "rule of three" and there are only two instances now.